src/containers/Header/BreadcrumbsRouter/Breadcrumbs.js

```diff
@@ -9,14 +9,21 @@ const Breadcrumbs = ({
 	routes, match, app, disableContainerBreadcrumbs
 }) => {
 	const { t } = useTranslation();
-	// Get new crumbs each time location has been changed
-	const crumbs = routes.filter(({ path }) => match.path.includes(path)).
+	const crumbs = routes.filter(({ path }) => {
+			if ((path.split("/").length - 1) === (match.path.split("/").length - 1)) { // Checking if the number of slash ('/') is the same
+				if (match.path.includes(path) && (path.match(/[^\/]+$/)[0] === match.path.match(/[^\/]+$/)[0])) { // I take the string after the last slash ('/') and check if they match each other
+					return path;
+				}
```

Comment:
@Aringoaway Add also else condition, since if the condition is not met, it will not return anything
```
} else {
	return match.path.includes(path);
}
```
alternatively, you can do this:

```
if (match.path.includes(path) && (path.match(/[^\/]+$/)[0] === match.path.match(/[^\/]+$/)[0])) {
	return path;
}
return match.path.includes(path);
```

Reply:
Ok, thanks for explanation